src/components/App.js.jsx

```diff
@@ -3,8 +3,23 @@ import { Link } from 'react-router'
 import '../css/App.css'
 import BuilderContainer from './containers/BuilderContainer.js.jsx'
 import DashboardContainer from './containers/DashboardContainer.js.jsx'
+import Ajax from './helpers/ajax.js'
+import { connect } from 'react-redux' 
 
 class App extends Component {
+
+  componentDidMount() {
+    const api = new Ajax()
+    api.fetchTags().then(response => this.storeTags(response.data))
```

Comment:
not necessary for such a readable and concise function, but for promises, it may be good to pass a named function to the promise callback instead of an anonymous function. That way the specific callback could be unit tested and against mocked data easily.